app.rb

```diff
@@ -268,6 +279,20 @@ def sp_logout_request
     redirect to(logout_request)
   end
 
+  def prepare_step_up_flow(session:, ial:, aal: nil)
+    if ial == 'step-up'
+      puts "\n    ~~~~~~~ prepare_step_up_flow"
```

Comment:
I think it's intended, but just to confirm, we want to log this, yeah? Not just leftover local debugging code?